spec/unit/provider/user/useradd_spec.rb

```diff
@@ -317,6 +317,28 @@
     end
   end
 
+  describe "#comment" do
+    before do
+      described_class.has_feature :libuser
+    end
+
+    let(:passwd_line) { "myuser:x:1492:1000:local comment:/home/myuser:/bin/bash" }
+
+    it "should return the local comment string when forcelocal is true" do
+      resource[:forcelocal] = true
+      allow(provider).to receive(:finduser).with('account', 'myuser').and_return(passwd_line)
+      expect(provider).to receive(:localcomment).and_return('local comment')
```

Comment:
I expect this should not be mocked as it will call finduser that is already mocked?